ecs-cli/modules/config/params.go

```diff
@@ -50,22 +51,24 @@ func recursiveFlagSearch(context *cli.Context, flag string) string {
 	}
 }
 
-// NewCliParams creates a new ECSParams object from the config file.
-func NewCliParams(context *cli.Context, rdwr ReadWriter) (*CliParams, error) {
-	ecsConfig, err := rdwr.GetConfig()
+// NewCLIParams creates a new ECSParams object from the config file.
+func NewCLIParams(context *cli.Context, rdwr ReadWriter) (*CLIParams, error) {
+	ecsConfig, configMap, err := rdwr.GetConfig()
+
 	if err != nil {
-		logrus.Error("Error loading config: ", err)
+		errors.Wrap(err, "Error loading config")
```

Comment:
This is not used at all. I think we can just return the error.Wrap here since this will be logged again during the execution.